### PR TITLE
Connect registration & login & register complete with veryfication

### LINKED ISSUE #73: connect authenticate proccess
Connect all authentication process.
All must work in dev server.

### PR DESCRIPTION
**Pull request by maryanchuk04**

### CODE REVIEW

File: StudCity.Frontend/src/pages/Authenticate/index.jsx [hunk 1]
```diff
@@ -31,31 +31,30 @@ function Authenticate() {
...
-							required={true}
 							value={formState.password}
 							setDisabled={setDisabled}
 							onChange={(event) => setFormState({ ...formState, password: event.target.value })}
 						/>
 						<a className="w-full text-center ml-1 font-medium text-base text-primaryAuthentication" href="">Forgot password</a>
 						<Button
-							disabled={disabled}
+							disabled={disabled || formState.email === "" || formState.password === ""}
+							className={"disabled:opacity-50"}
```
Comment (Serhiy-dotcom) [inline StudCity.Frontend/src/pages/Authenticate/index.jsx]: you can remove curly brackets here

File: StudCity.Frontend/src/pages/Registration/index.jsx [hunk 2]
```diff
@@ -84,8 +85,8 @@ function Registration() {
 							validate={passwordMatchValidation}
 						/>
 						<Button
-							disabled={disabled}
-							className={"bg-primaryRegistration"}
+							disabled={disabled || formState.email === "" || formState.password === "" || formState.confirmPassword === ""}
+							className={"bg-primaryRegistration disabled:opacity-50"}
```
Comment (Serhiy-dotcom) [inline StudCity.Frontend/src/pages/Registration/index.jsx]: same here

File: StudCity.Frontend/src/pages/Verify-email/index.jsx [hunk 3]
```diff
@@ -2,29 +2,41 @@ import React, { useState } from "react";
 import TextField from "../../UI/fields/TextField";
 import { numberValidation } from "../../utils/validators/validators";
 import Svg from "../../components/Svg";
+import { AuthenticateService } from "../../services/authenticateService";
+import { useParams, useNavigate } from "react-router-dom";
 
 function VerifyEmail() {
-	const [verifyNumbersArray, setVerifyNumbersArray] = useState([ "", "", "", "", "", "" ]);
+	const service = new AuthenticateService();
+	const accountId = useParams().accountId;
```
Comment (Serhiy-dotcom) [inline StudCity.Frontend/src/pages/Verify-email/index.jsx]: I would suggest to write like this: 
const { accountId } = useParams();

File: StudCity.Frontend/src/pages/Verify-email/index.jsx [hunk 4]
```diff
@@ -2,29 +2,41 @@ import React, { useState } from "react";
...
+	const navigate = useNavigate();
+
+	const [verifyNumbersArray, setVerifyNumbersArray] = useState(["", "", "", "", "", ""]);
```
Comment (Serhiy-dotcom) [inline StudCity.Frontend/src/pages/Verify-email/index.jsx]: refactor to using simple string instead

**Review by Serhiy-dotcom — CHANGES_REQUESTED** [review]

**Review by Serhiy-dotcom — APPROVED** [review]

**Review by Vova-Romaniuk — APPROVED** [review]

**Review by Serhiy-dotcom — APPROVED** [review]

**Review by Vova-Romaniuk — APPROVED** [review]

**Pull request merged**